src/kaocha/hierarchy.clj

```diff
@@ -8,6 +8,11 @@
   [tag parent]
   (alter-var-root #'hierarchy derive tag parent))
 
+(defn underive!
+  "Add a parent/child relationship to kaocha's keyword hierarchy."
```

Comment:
This removes a parent/child relationship, right?

Reply:
yes

Reply:
Then I'd suggest:

```suggestion
  "Remove a parent/child relationship from Kaocha's keyword hierarchy."
```

Reply:
Are you planning to make this change?